agent_build_refactored/managed_packages/steps/build_dev_requirements.sh

```diff
@@ -36,12 +36,24 @@ ldconfig
 cd ~
 export PATH="/usr/local/bin:/root/.cargo/bin:${PATH}"
 export PKG_CONFIG_PATH="/usr/local/lib64/pkgconfig:/usr/local/lib/pkgconfig:${PKG_CONFIG_PATH}"
-curl --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y --default-toolchain "${RUST_VERSION}"
+
```

Comment:
Why switch from rustup?  (Out of curiosity)

Reply:
Newer versions of Rust, including rustup, have already dropped the support of the glibc < "2.17", and we still support glibc "2.12", so newest rustup just can not start :(

That may sound as a good signal for us to also drop so old glibc versions, but I think this release is already has enough changes and it would be better to do it later.